drv/vsc85xx/src/vsc8562.rs

```diff
@@ -110,28 +203,83 @@ impl<'a, 'b, P: PhyRw> Vsc8562Phy<'a, 'b, P> {
 
         // "Fix for bz# 21484 ,TR.LinkDetectCtrl = 3"
         self.fix_bz21484()?;
+
+        // ...and we're done with phy_reset_private!
+
+        // Note that the Sidecar BSP performs additional SERDES tuning
+        // in `vsc7448_postconfig`, based on empirical testing.
         Ok(())
     }
 
-    fn fix_bz21484(&mut self) -> Result<(), VscError> {
-        self.phy.broadcast(|v| {
-            v.write(phy::TR::TR_16(), 0xa7f8.into())?;
-            v.modify(phy::TR::TR_17(), |r| {
-                r.0 &= 0xffe7;
-                r.0 |= 3 << 3;
-            })?;
-            v.write(phy::TR::TR_16(), 0x87f8.into())?;
+    /// Modifies the SERDES6G CFG register
+    pub fn tune_sd6g_ob_cfg(&mut self, cfg: Sd6gObCfg) -> Result<(), VscError> {
+        cfg.check_range()?;
+        self.mcb_read(0x3f, 0)?;
+        self.sd6g_ob_cfg_write(
+            cfg.ob_ena1v_mode,
+            cfg.ob_pol,
+            cfg.ob_post0,
+            cfg.ob_post1,
+            cfg.ob_sr_h,
+            cfg.ob_resistor_ctr,
+            cfg.ob_sr,
+        )?;
+        self.mcb_write(0x3f, 0)?;
+        Ok(())
+    }
 
-            // "Fix for bz# 21485 ,VgaThresh100=25"
-            v.write(phy::TR::TR_16(), 0xafa4.into())?;
-            v.modify(phy::TR::TR_18(), |r| {
-                r.0 &= 0xff80;
-                r.0 |= 25
-            })?;
-            v.write(phy::TR::TR_16(), 0x8fa4.into())
+    pub fn read_sd6g_ob_cfg(&mut self) -> Result<Sd6gObCfg, VscError> {
+        self.mcb_read(0x3f, 0)?;
+        let v = self.macsec_csr_read(7, 0x28)?;
+        Ok(Sd6gObCfg {
+            ob_ena1v_mode: ((v >> 30) & 1) as u8,
+            ob_pol: ((v >> 29) & 1) as u8,
+            ob_post0: ((v >> 23) & 63) as u8,
+            ob_post1: ((v >> 18) & 5) as u8,
```

Comment:
This isn't a power of two minus one, is that deliberate? (Generally speaking I'd encourage the use of either binary literals or explicit shifts for masks like this.)